helm/aws-load-balancer-controller/values.yaml

```diff
@@ -87,6 +87,11 @@ enableCertManager: false
 # ingresses without ingress class annotation and ingresses of type alb
 ingressClass: alb
 
+# The ingressClassParams the ingress class refers to and enforce settings for a set of Ingresses when using with ingress Controller.
+ingressClassParams:
```

Comment:
why not create IngressClassParams resource and refer from the IngressClass instead? users will be able to specify the ingress class params spec during chart installation

Reply:
since the params spec can be user specific and vary, would that be better to let users create ingress class params resources and just refer their name in charts during install/upgrade? That could be easier to maintain and less error-prone?